src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/d2575abd28.js" crossorigin="anonymous"></script>
+    <title>Movie App</title>
+</head>
+<body>
+  <header id="header" class="show">
+    <div class="logoDiv">
+      <img id="logo" src="./images/logo1.jpg" alt="image-logo" srcset="">
+    </div>
+      <nav id="nav">
+          <nav id="navbar">
+              <a class="nav-link" href="">Home</a>
+              <span>
+                  <a href="">Movies Show</a>
+                  <span id="counter"></span>
+              </span>
+          </nav>
+      </nav>
+  </header>
+  <main>
+     <section id="movies-container" class="show">
+     </section>
+     <section id="popup-window" class="hide">
+     </section>
+     <div id="overlay" class="hide"></div>
+  </main>
+  <footer id="footer" class="show">
+      <p class="footerP">Created by <a href="">Zohra Neda</a>  & <a href=""> Nahid Raihan</a></p>
+  </footer>
+</body>
+</html>
```

Comment:
- [ ] [ OPTIONAL] You should try and make your app responsive 
![Screenshot from 2023-06-02 19-54-15](https://github.com/Zohra-Neda/JavaScript-Capstone-Project/assets/66097371/57b0a0b0-6376-40d4-8109-3bb6054b0c21)